test/controllers/issueish-detail-controller.test.js

```diff
@@ -109,6 +109,15 @@ describe('IssueishDetailController', function() {
       assert.isFalse(op1.isEnabled());
       assert.strictEqual(op1.getMessage(), 'Rebase in progress');
     });
+    it('is disabled if pullRequest.headRepository is null', function() {
+      const props = issueishDetailControllerProps({}, {});
+      props.repository.pullRequest.headRepository = null;
```

Comment:
I...I don't love this, but also, the way the test factories are currently set up, you can't supply a parameter to our test factories that makes an entire nested part of the Relay response null.   We should probably come up with a better pattern for these factories.